students/jjakub/session07/html_render/test_html_render.py

```diff
@@ -263,12 +264,13 @@ def test_indent():
     html = Html("some content")
     file_contents = render_result(html, ind="   ").rstrip()  #remove the end newline
 
-    # print(file_contents)
     lines = file_contents.split("\n")
     print(file_contents)
-    # assert lines[0].startswith("   <")
+    # assert lines[0].startswith("   <") 
+        ### Question: The item at line[0] is the docheader, why would there be a indent before that?
```

Comment:
That's because the `file_contents = render_result(html, ind="    ").rstrip()` was made with `ind="    "`, which is to be the base indentation level. I suppose you might have not handled that?